lib/payroll_hero/api/client.rb

```diff
@@ -8,10 +8,11 @@ module Api
     class Client
       attr_reader :client
 
-      def initialize(token, base_url)
+      def initialize(token, base_url, cookies = {})
         headers = {
           'User-Agent' => user_agent_string,
         }
+        headers.merge!({ 'Cookie' => cookies.map { |key, value| "#{key}=#{value}" }.join('; ') }) unless cookies.nil? || cookies.empty?
```

Comment:
is this cookie secure? signed?

could it get manipulated by simply editing it "superadmin = true" ?

Reply:
@mathieujobin This gem contains only the pass-through mechanism to attaches cookies to server-to-server API calls as if it were browser-to-server. But yes, the cookie for superadmin (set by core and used by LM) is secure & signed to prevent tampering